src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -55,11 +55,11 @@ private void Dynamic_All(TraceEvent obj)
                 ICounterPayload payload;
                 if (payloadFields.ContainsKey("CounterType"))
                 {
-                    payload = payloadFields["CounterType"].Equals("Sum") ? (ICounterPayload)new IncrementingCounterPayload(payloadFields) : (ICounterPayload)new CounterPayload(payloadFields);
+                    payload = payloadFields["CounterType"].Equals("Sum") ? (ICounterPayload)new IncrementingCounterPayload(payloadFields, _interval) : (ICounterPayload)new CounterPayload(payloadFields);
                 }
                 else
                 {
-                    payload = payloadFields.Count == 6 ? (ICounterPayload)new IncrementingCounterPayload(payloadFields) : (ICounterPayload)new CounterPayload(payloadFields);
+                    payload = payloadFields.Count == 6 ? (ICounterPayload)new IncrementingCounterPayload(payloadFields, _interval) : (ICounterPayload)new CounterPayload(payloadFields);
```

Comment:
This lower branch is only generated by runtimes earlier than Preview7 right? If so it would be fine to eliminate it.

Reply:
Will do in a future PR :)